reader.py

```diff
@@ -0,0 +1,38 @@
+import http.client
+import json
+import datetime
+from util import SortUtil
+
+#Reads the data from the resources
+class Reader:
```

Comment:
It's better that the class has a specific name such as ArticleReader, NewsReader or HttpResponseReader. What does this class read? What is the purpose of this class?

Reply:
Class name has been changed and the purpose of the class has been written in the comment